### DISCUSSION TITLE
Open for hiring

### DISCUSSION THREAD

**Discussion by haplle** [community/community#131203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

is there any hiring ?

**Comment by ghostinhershell** [comment]
Hi @haplle, we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
